pkg/icingaredis/heartbeat.go

```diff
@@ -62,6 +63,11 @@ func (h *Heartbeat) LastReceived() int64 {
 	return atomic.LoadInt64(&h.lastReceivedMs)
 }
 
+// LastMessage returns the last message's time in ms.
+func (h *Heartbeat) LastMessage() int64 {
```

Comment:
Since this does not return the actual heartbeat message, it should be named according to what it actually returns.

Reply:
This method name tries to follow `LastReceived`, returning the last received timestamp while `LastMessage` returns the last message's timestamp. Thus, what would you suggest then? Especially when fitting with its twin method.

Reply:
> while `LastMessage` returns the last message's timestamp. Thus, what would you suggest then?

Please don't take this the wrong way, and I'm not trying to impose my own ideals on you. I just think that `LastMessage` is not the same as `last message timestamp`, and I apologize for not providing an alternative name from the beginning that I'm thinking of. Thus, my first suggestion would be to simply call it `LastMessageTime[stamp]`.